src/main.rs

```diff
@@ -82,15 +83,63 @@ fn main() -> Result<(), Error> {
         Network::homechain(
             home_ws.clone(),
             &settings.relay.account,
-            &*consul_configs::wait_or_get("homechain", "nectar_token_address"),
-            &*consul_configs::wait_or_get("homechain", "erc20_relay_address"),
+            &*consul_configs::wait_or_get(
```

Comment:
`wait_or_get` and `create_contract_abi` receive `clippy` warnings here but I'd say you should ignore it.

If you decide to tackle this linting, I'd personally consider doing so through the refactor proposed in the review rather than doing so directly.

Reply:
Cool, where's the proposed refactor? I'll probably add that handy directive. I'm having trouble running clippy locally.

Reply:
Check out https://github.com/polyswarm/relay/tree/fix/test-refactor